src/server/nyplApiClient/index.ts

```diff
@@ -22,13 +22,16 @@ export class NyplApiClientError extends Error {
   }
 }
 
-const nyplApiClient = async (options = { apiName: "platform" }) => {
-  const { apiName } = options
+const nyplApiClient = async ({
+  apiName = "platform",
+  version = "v0.1",
+} = {}) => {
   if (CACHE.clients[apiName]) {
```

Comment:
can you update this cache to include the version? otherwise i fear this may return a cached v0.1 client when you want a v0.3